docs/topics/auth/default.txt

```diff
@@ -763,6 +763,48 @@ email in the desired domain and if not, redirects to the login page::
 
 .. currentmodule:: django.contrib.auth.decorators
 
+The ``login_not_required`` decorator
```

Comment:
Let's move both of these to be just under `The ``LoginRequiredMixin`` mixin` section